packages/react/src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -20,6 +20,7 @@ export const ButtonGroup = forwardRef(
             size: buttonSize,
             as: buttonsAs,
             variant: index === 0 ? 'primary' : 'secondary',
+            hasArrow: index === 0 ? true : false,
```

Comment:
FYI, we recently had to update this in PVC to support overwrite, because some designs required both buttons in a group to have arrows.

Reply:
This supports overwrite but it takes away the default behavior which is to have a arrow.